.devcontainer/cuda12.2-gcc12/devcontainer.json

```diff
@@ -7,12 +7,10 @@
   "initializeCommand": [
     "/bin/bash",
     "-c",
-    "mkdir -m 0755 -p ${localWorkspaceFolder}/.{aws,cache,config}"
+    "mkdir -m 0755 -p ${localWorkspaceFolder}/.{cache,config,sccache}"
   ],
   "containerEnv": {
-    "SCCACHE_REGION": "us-east-2",
-    "SCCACHE_BUCKET": "rapids-sccache-devs",
-    "VAULT_HOST": "https://vault.ops.k8s.rapids.ai",
+    "SCCACHE_DIR": "${containerWorkspaceFolder}/.sccache",
```

Comment:
I left this, in the PR, not because I think it should be merged(obviously) but rather to highlight how sccache can be used locally for contributors like myself. I am unable to access the cloud o bviously since I'm not an NVIDIA employee. 

Maybe you can comment? @jrhemstad

Reply:
sccache works the same as with the cloud, it just stores aa local cache. So you need to populate it once but thats it

Reply:
Per our discussion, here is the issue https://github.com/NVIDIA/cccl/issues/593